components/builder-web/app/statuspage/_statuspage-indicator.component.scss

```diff
@@ -0,0 +1,16 @@
+.color-dot {
```

Comment:
One thing I didn't notice until now is that these styles are being defined globally. While a collision is unlikely, it'd be good to follow the existing pattern of scoping component styles underneath either the element name (`hab-statuspage`) or giving the component template a class of `statuspage-component` and defining the selectors under that.